requirements/requirements_docs.txt

```diff
@@ -5,3 +5,4 @@ sphinx-hoverxref==1.1.3
 sphinx-autobuild~=2021.3
 sphinxcontrib-towncrier==0.2.1a0
 towncrier==21.9.0
+sphinx-reredirects==0.1.1
```

Comment:
is this being used?

Reply:
sorry stackoverflow solution artifact

Reply:
Done in [`d8070db`](https://github.com/DisnakeDev/disnake/pull/722/commits/d8070db1f45fdb46fc3264db52d117c2ad5e86f6)